examples/AlwaysSucceeds.purs

```diff
@@ -28,9 +28,15 @@ import Contract.TxConstraints as Constraints
 import Contract.Value as Value
 import Contract.Wallet (mkNamiWalletAff)
 import Data.BigInt as BigInt
+import Examples.MultipleRedeemers as MultipleRedeemers
 
 main :: Effect Unit
 main = launchAff_ $ do
+  alwaysSucceedsContract
+  MultipleRedeemers.threeRedeemerContract
```

Comment:
I don't think this should be added to `AlwaysSucceeds`. You can just add a `main` to `MultipleRedeemers` instead. To run it, you can edit the `ps-entrypoint` variable in the Makefile

Reply:
I'll add that as documentation to the main function.